<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Script</class>
 <widget class="QWidget" name="Script">
  <property name="windowTitle">
   <string>Script</string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>262</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>14</number>
   </property>
   <property name="topMargin">
    <number>14</number>
   </property>
   <property name="rightMargin">
    <number>14</number>
   </property>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="openPB">
       <property name="text">
        <string>Open Script</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runPB">
       <property name="text">
        <string>Run JS Script</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePB">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>8</number>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QGroupBox" name="displayGB">
      <property name="title">
       <string>Display Script</string>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>openPB</tabstop>
  <tabstop>runPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
